src/main/java/uk/gov/pay/adminusers/resources/ServiceUpdateOperationValidator.java

```diff
@@ -143,7 +141,7 @@ public ServiceUpdateOperationValidator(RequestValidations requestValidations) {
     }
 
     private List<String> validateServiceNameValue(JsonNode operation, String path) {
-        boolean allowEmpty = !FIELD_NAME.equals(path) && !path.endsWith('/' + SupportedLanguage.ENGLISH.toString());
+        boolean allowEmpty = !path.endsWith('/' + SupportedLanguage.ENGLISH.toString());
```

Comment:
should the `path.endsWith` not check for `SupportedLanguage.WELSH.toString()` as well?

Reply:
No, the validation rules are subtly different. We don’t allow an empty string to be set as the English-language service name. However, we do allow an empty string to be set as the Welsh-language service name. It’s our way of letting someone remove a Welsh-language service name if they decide they don’t want one any more (there’s logic elsewhere to ignore empty strings for the translated service names when building service objects).